packages/jest-util/src/notEmpty.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default function notEmpty<T>(value: T | null | undefined): value is T {
+  return value !== null && value !== undefined;
```

Comment:
What about the following rewrite:

```suggestion
export default function isNonNullable<T>(value: T): value is NonNullable<T> {
  return value != null;
```